Sources/Testcontainers/Strategies/Docker+ClientStrategy.swift

```diff
@@ -0,0 +1,73 @@
+//
+//  Docker+ClientStrategy.swift
+//  Testcontainers
+//
+//  Created by cristian on 20/08/24.
+//
+
+import Foundation
+import Logging
+import NIO
+
+final class DockerClientStrategy {
+    
+    var strategies: [DockerClientStrategyProtocol] = [
+        TestcontainersStrategy(),
```

Comment:
looks like it is not working. Desktop app is initialized and the connection is already stablished but tests can not run against it

Reply:
Pull and try again